### PR TITLE
Branch docs for Katello 4.1

### PR DESCRIPTION
**Pull request by jeremylenz**
Includes https://github.com/theforeman/foreman-documentation/pull/515 (this should be merged after that)

### CODE REVIEW

**Comment by lzap** [comment]
TY, can you also update the .github/PULL_REQUEST_TEMPLATE.md ?

**Comment by lzap** [comment]
I merged F2.5 PR can you rebase please?

**Review by lzap — APPROVED** [review]
TY

**Comment by lzap** [comment]
Oh it is still a draft, okay.

**Comment by jeremylenz** [comment]
@lzap yeah, still working on the release notes! :grin:

**Comment by lzap** [comment]
Is this ready now? I see RC2 in the text, I am fine with that just let me know it is ready (I see you assigned the label).

**Comment by jeremylenz** [comment]
@lzap It's ready for review now, just want to get someone from my team to take a look before it's merged :)

@jsherrill @jturel @jjeffers can at least one of you pls let me know if the release notes look okay!

Thanks all!

**Comment by jjeffers** [comment]
> @lzap It's ready for review now, just want to get someone from my team to take a look before it's merged :)
> 
> @jsherrill @jturel @jjeffers can at least one of you pls let me know if the release notes look okay!
> 
> Thanks all!

Looks fine - although I wonder if the list of changes seems to be just reproducing the changelog entries. I thought the release notes were more descriptive about highlighted changes? I could be wrong!

**Comment by jturel** [comment]
> > @lzap It's ready for review now, just want to get someone from my team to take a look before it's merged :)
> > @jsherrill @jturel @jjeffers can at least one of you pls let me know if the release notes look okay!
> > Thanks all!
> 
> Looks fine - although I wonder if the list of changes seems to be just reproducing the changelog entries. I thought the release notes were more descriptive about highlighted changes? I could be wrong!

Agree I think the outlined features part looks good but I think providing a link to the CHANGELOG is sufficient (rather than placing the whole content of it on the site)

**Comment by jeremylenz** [comment]
@jturel agree, updated :)

**Review by jturel — APPROVED** [review]
ACK

**Comment by jeremylenz** [comment]
Remember to squash when merging - or I can squash if needed

File: web/content/2.5.md [hunk 1]
```diff
@@ -306,15 +306,28 @@ Puppet 5 is officially End of Life. Foreman {{page.version}} still supports Pupp
...
+## Deprecations
+* API endpoints for katello-agent actions have been deprecated. Consider switching to remote execution.
+* New deprecation warning on the Register Content Host page - Use Global Registration instead (/hosts/register)
 
-### Contributors
+
+### Contributors to Foreman 2.5
 * Adam Ruzicka
 * Adi Abramovich
 * Adi Abramovitch
```
Comment (melcorr) [inline web/content/2.5.md:333]: Are there two people?

**Review by melcorr — COMMENTED** [review]
Adi is listed twice, but otherwise am happy to merge this.

File: web/content/2.5.md (hunk 1 above)
Comment (jeremylenz) [reply web/content/2.5.md:333]: Probably not ;)  @adiabramovitch Which is the correct spelling?

Comment (adiabramovitch) [reply web/content/2.5.md:333]: Adi Abramovich (:

**Comment by lzap** [comment]
I would not fix this, honestly, Adi had to have a typo in github.com and since this list is auto-generated and we will not rebase full history of our git just to correct a name, then we must live with this :-)

```
$ git shortlog --summary --numbered --email | grep Abram
     2	Adi Abramovitch <57755873+adiabramovitch_users.noreply.github.com>
     1	Adi Abramovich <aabramov_redhat.com>
```

We have more of these, many more:

```
$ git shortlog --summary --numbered --email | grep Luk
   328	Lukas Zapletal <lzap+git_redhat.com>
    38	Lukáš Zapletal <lzap_redhat.com>
    17	Lukáš Zapletal <lzap+git_redhat.com>
     2	Lukas Zapletal <lzap_redhat.com>
```

**Comment by jeremylenz** [comment]
@melcorr Removed the duplicate name. @lzap what can I say, I'm nice :man_shrugging:

**Review by melcorr — APPROVED** [review]

**Comment by lzap** [comment]
> I'm nice

Kudos, that's great. I commented on that because I thought this is gonna be replaced on the next release but now I get that the typo might not be in the next version actually. Makes sense, thanks!

**Pull request merged**